client/src/components/common/request-details.vue

```diff
@@ -1,23 +1,40 @@
 <template>
   <div>
-    <div style="display: flex; justify-content: space-between; flex-flow: row wrap; margin-bottom: 15px">
-      <div style="margin: 30px 30px 0">
+    <div style="display: flex; justify-content: space-between; flex-flow: row wrap; margin-bottom: 5px">
+      <div style="margin: 30px 0 -2px">
```

Comment:
While you are here, please change the inline style to a "class" and take advantage of the Vuetify helper classes.

https://vuetifyjs.com/en/styles/flex/
https://vuetifyjs.com/en/styles/spacing/

Reply:
Lines 3 and 4.